crates/rpc/rpc/src/eth/api/fees.rs

```diff
@@ -119,6 +120,8 @@ where
             last_header.gas_used,
             last_header.gas_limit,
             last_header.base_fee_per_gas.unwrap_or_default(),
+            self.provider().chain_spec().base_fee_change_denominator,
+            self.provider().chain_spec().elasticity_multiplier,
```

Comment:
these lines unnecessarily clone the `Arc` twice. even though it's cheap, i'd prefer to avoid this by doing
```rust
let chain_spec = self.provider().chain_spec();
base_fee_per_gas.push(U256::from(calculate_next_block_base_fee(
    last_header.gas_used,
    last_header.gas_limit,
    last_header.base_fee_per_gas.unwrap_or_default(),
    chain_spec.base_fee_change_denominator,
    chain_spec.elasticity_multiplier,
)));

Reply:
done